deployment/cdk/opensearch-service-migration/lib/network-stack.ts

```diff
@@ -48,7 +48,6 @@ export interface NetworkStackProps extends StackPropsExt {
     readonly elasticsearchServiceEnabled?: boolean;
     readonly captureProxyServiceEnabled?: boolean;
     readonly targetClusterProxyServiceEnabled?: boolean;
-    readonly migrationAPIEnabled?: boolean;
```

Comment:
Did we not use this anywhere?

Reply:
Nope, we've got some hard to catch unused values in the CDK because of bi-directional references.